### ISSUE TITLE
Changing user

### ISSUE THREAD

**Issue by Silverunit** [danpros/htmly#168]
Hi again,
Quick question. If I would like to change the user that are now admin for the posts.
How do I do that? and is it even possible? 

For my test I used a simple name and password. Now I want to make it live and need stronger login name and password.

Best regards

**Comment by danpros** [comment]
Yes that is possible let say you have `admin.ini` (username: `admin`) and in content folder you will find `admin` folder. To change the username than just rename `admin.ini` becomes `example.ini` and rename the folder become `example`. You can login using `example` user.

To secure the login page you can using https://www.google.com/recaptcha

**Comment by Silverunit** [comment]
Perfect, I was thinking that was the case but was not sure.
Yes I saw that you made captcha available for the login in the update. Thats even more secure.

And how about the password?
Is it changeable in this way also? I noticed that it was decrypted in some way.

**Comment by danpros** [comment]
Yes you can change it like that.

Security? Well you can read this part https://github.com/danpros/htmly#making-a-secure-password

But clear password+recaptcha is fine as long as you have strong password. WordPress save their db password on plain text either isn't? but on HTMLy, folder that contains those data protected by .htaccess.

Seems you change the recent posts lists. On HTMLy all options is in config.ini and to call it simply:

````
config('your.options');
````
I will creating new options for this as the default:

````
recent.count
```

**Comment by Silverunit** [comment]
Hi, tried to change admin.ini to "something" and the user folder to "something". But I lost all posts.
And then got an error when viewing post. Do I also have to change the role=" " in admin.ini?

And also created a reCaptcha but I got invalid key for domain. Have added
with and with out www for both the test domain and live domain. So that should not be the issue.

Removed it and changed back and now its like it used to be :)
